From 26051aa87e50e075e3acd87c9a698f2026efb5f7 Mon Sep 17 00:00:00 2001
From: Michal Vasilek <michal@vasilek.cz>
Date: Wed, 6 Jul 2022 12:47:04 +0200
Subject: [PATCH] build: fix docs cross build

gi-docgen is a binary that runs on the host, so the meson should search
for it on the host
---
 meson.build | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b5dbea53a..1e3115f05 100644
--- a/meson.build
+++ b/meson.build
@@ -169,7 +169,8 @@ m_dep = cc.find_library('m')
 
 gidocgen_dep = dependency('gi-docgen', version: '>= 2021.1',
 			  fallback: ['gi-docgen', 'dummy_dep'],
-			  required: get_option('gtk_doc'))
+			  required: get_option('gtk_doc'),
+			  native: true)
 
 # Although GTK+ 3.10 includes hi-dpi functionality, it does not require a cairo with
 # cairo_surface_set_device_scale(), which we also need if we're to support hi-dpi,
-- 
GitLab

